### ISSUE TITLE
_before.groovy:31-33: Add a new class FkSkSession that...

### ISSUE THREAD

**Issue by 0pdd** [zerocracy/farm#1139]
The puzzle `89-f08c5086` from #89 has to be resolved:

https://github.com/zerocracy/farm/blob/499a6e169756ee097d71e6151b22d64c640a8331/src/test/resources/com/zerocracy/bundles/notify_in_slack_without_user/_before.groovy#L31-L33

The puzzle was created by rultor on 11-Jun-18. 

Estimate: 30 minutes,     

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#1139](https://github.com/zerocracy/farm/issues/1139) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#1139](https://github.com/zerocracy/farm/issues/1139) assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1831013958); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by krzyk** [comment]
@0crat refuse

**Comment by 0crat** [comment]
> @0crat refuse ([here](https://github.com/zerocracy/farm/issues/1139#issuecomment-402791510))

@krzyk The user @krzyk[/z](https://www.0crat.com/u/krzyk) resigned from [#1139](https://github.com/zerocracy/farm/issues/1139), please stop working. Reason for job resignation: Order was cancelled

**Comment by 0crat** [comment]
Tasks refusal is discouraged, see [§6](http://www.zerocracy.com/policy.html#6): -15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
The job [#1139](https://github.com/zerocracy/farm/issues/1139) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1971157137); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by paulodamaso** [comment]
@0crat wait, this task depends on #1140

**Comment by 0crat** [comment]
> @0crat wait, this task depends on #1140  ([here](https://github.com/zerocracy/farm/issues/1139#issuecomment-406628662))

@paulodamaso Can't understand "wait,", try one of these:
  - `assign` Assign a performer to this issue
  - `boost` Set a boost for the job
  - `continue` Remove a job's impediment
  - `hello` Just say hello
  - `in` Register this issue in scope (WBS)
  - `out` Close the order and remove this job from scope
  - `pay` Pay a user some extra cash
  - `quality` Review a task
  - `resign` Resign from this issue
  - `status` Check the status of the job
  - `version` Print current version of the bot
  - `wait` Register an impediment for a job

**Comment by paulodamaso** [comment]
@0crat wait this task depends on #1140

**Comment by 0pdd** [comment]
The puzzle `89-f08c5086` has disappeared from the source code, that's why I closed this issue.

**Comment by 0pdd** [comment]
@0pdd the puzzle [#1544](https://github.com/zerocracy/farm/issues/1544) is still not solved.

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@7bda313d-71fb-41d0-8d6f-eda5801ea74b, thread: pool-253-thread-1
java.lang.NumberFormatException[65] java.lang.NumberFormatException: For input string: "2018-08-03T16:40:10.596Z"
```

[0.28.15](https://github.com/zerocracy/farm): Issue: zerocracy/farm#1139, Action: `closed`

**Comment by paulodamaso** [comment]
@g4s8 0crat failed to close, please close it

**Comment by krzyk** [comment]
@0crat out

**Comment by 0crat** [comment]
@krzyk[/z](https://www.0crat.com/u/krzyk) the issue is closed not by @0pdd[/z](https://www.0crat.com/u/0pdd) (its creator); I won't close the order; please, re-open it and ask @0pdd[/z](https://www.0crat.com/u/0pdd) to close it

**Comment by paulodamaso** [comment]
@g4s8 0pdd closed the issue, but 0crat failed removing it from wbs; please remove it.

**Comment by paulodamaso** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@0crat out

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/zerocracy/farm/issues/1139#issuecomment-411532983))

@g4s8 There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@ce1bb5b2-f136-41eb-a7c2-a85674933de4, thread: AsyncProc-6
com.jcabi.xml.StrictXML[124] java.lang.IllegalArgumentException: 1 error(s) in XML document: -1:-1: Duplicate unique value [gh:zerocracy/farm#1139] declared for identity constraint of element "verbosity".
```

[0.29.8](https://github.com/zerocracy/farm): CID: [2201959858](https://www.0crat.com/C3NDPUA8L/2201959858), Type: "Close job", Author: "g4s8"

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
> @0crat out ([here](https://github.com/zerocracy/farm/issues/1139#issuecomment-411532983))

@g4s8 The job [#1139](https://github.com/zerocracy/farm/issues/1139) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by krzyk** [comment]
@g4s8 I don't see payment for this job

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/1139#issuecomment-411720342))

@g4s8 This is what I know about this job in [C3NDPUA8L](https://www.0crat.com/p/C3NDPUA8L), as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1139](https://github.com/zerocracy/farm/issues/1139) is not in scope
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%231139%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by g4s8** [comment]
@0crat pay 35min @paulodamaso

**Comment by 0crat** [comment]
Direct payment from @g4s8[/z](https://www.0crat.com/u/g4s8): +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Direct payment to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) in #1139, which is discouraged, see [§49](http://www.zerocracy.com/policy.html#49): -20 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0pdd** [comment]
@0pdd 3 puzzles [#2282](https://github.com/zerocracy/farm/issues/2282), [#2283](https://github.com/zerocracy/farm/issues/2283), [#2284](https://github.com/zerocracy/farm/issues/2284) are still not solved; solved: [#1544](https://github.com/zerocracy/farm/issues/1544).
